src/components/login/Login.js

```diff
@@ -46,17 +66,25 @@ const Login = () => {
       );
 
       navigate("/auth/sign-up");
+      return;
     }
 
-    if (data.result === "재로그인이 필요한 유저입니다") {
+    if (data.result === RESPONSE_MESSAGE.RELOGIN_REQUIRED) {
       navigate("/auth/login");
+      return;
     }
-  };
 
-  useQuery("loginData", loginData, {
-    enabled: !!userEmail,
-    onSuccess,
-  });
+    if (data.error) {
+      if (data.error.code === 400) {
+        setErrorMessage(ERROR_MESSAGE.BAD_REQUEST);
+        return;
+      }
```

Comment:
```suggestion
      if (data.error.code === 400) {
        setErrorMessage(ERROR_MESSAGE.BAD_REQUEST);
      }
```

에러 상태 코드는 하나만 존재하니까 중간에 return문 지워주셔도 될 것 같다는 생각이 듭니다-!

Reply:
아래로 굳이 내려가지 않게 할 수 있으면 return 해주는게 좋다고 생각해서 저렇게 작성했어요~ 제 생각에는 최대한 컨트롤할 수 있을 만큼 하는게 좋을 것 같은데 서영님 의견은 어떠실까요?

Reply:
움,,전 그동안 따로 작성해주지 않았었는데요, 성훈님 이야기를 들어보니 써주는 것이 맞는 것 같은 느낌도 들어요~  
근데 혹시 아래의 500에러의 경우 작성해주지 않으신건 마지막 부분이라 더이상 내려가지 않기 때문이신가요? ?

Reply:
넵 서영님께서 말씀해주신 이유가 맞습니다..!